### PR TITLE
Allow for specifying config with env vars

### PR DESCRIPTION
**Pull request by vdbg**
can specify config both through `config.toml` and environment variables.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    def load(self) -> dict[str, dict]:
+        ret = self.__load__("template." + self._file)
+        if not ret:
+            raise Exception(f"File template.{self._file} required.")
+
+        # overwrite template with config, if exists
+        config = self.__load__(self._file)
+        if config:
+            for k, v in config.items():
```
Comment (Nickolaim) [inline config.py]: dictionary has dict.update that looks like does what you want: https://docs.python.org/3.8/library/stdtypes.html?highlight=dict.update#dict.update

File: main.py [hunk 2]
```diff
@@ -1,62 +1,56 @@
...
+        f"Python version {SUPPORTED_PYTHON_MAJOR}.{SUPPORTED_PYTHON_MINOR} or later required. Current version: {platform.python_version()}."
+    )
 
 try:
-    config = get_config()
+    config = Config("config.toml", "myair_influx").load()
     main_conf = config["main"]
     logging.getLogger().setLevel(logging.getLevelName(main_conf["logverbosity"]))
     sleep_time = main_conf["loop_minutes"] * 60
+    logging.debug(f"CONFIG: {config}")
```
Comment (Nickolaim) [inline main.py:29]: Logging secrets?

**Review by Nickolaim — APPROVED** [review]

File: main.py (hunk 2 above)
Comment (vdbg) [reply main.py:29]: if you want debug logging, you get debug logging

**Review by Nickolaim — APPROVED** [review]

File: config.py (hunk 1 above)
Comment (vdbg) [reply config.py]: good one! Thanks :)

**Pull request merged**